src/dev/shared/DevToys.Shared/Core/Assumes.cs

```diff
@@ -35,5 +35,13 @@ public static void IsTrue(bool value, string argumentName)
                 throw new Exception($"'{argumentName}' is expected to be True");
             }
         }
+
+        public static void NotZeroOrBelow(long value, string argumentName)
```

Comment:
If I'm not wrong, you wanted to use this method to check an argument. If so, can you please move this method to `Arguments` class instead?

Reply:
Done.